lib/index.js

```diff
@@ -1,35 +1,55 @@
 const playerNameInputs = document.querySelectorAll('.player-name-inputs');
 const domPlayerDivs = document.querySelectorAll('.player-names');
 const domCategoryDivs = document.querySelectorAll('.category-text');
+const ddWagerInput = document.querySelector('.wager-dd-text-input');
+const wagerErrorText = document.querySelector('.dd-wager-error-message');
+const answerInput = document.querySelector('.answer-text-input');
+const answerResponseText = document.querySelector('.answer-message-text');
 
-// store these in game class? having trouble accessing them here.
+// store these in classes. having trouble accessing them here.
 let currentClue;
 let currentRoundObj = {};
 
+let game = new Game();
+
 document.querySelector('.restart-button').addEventListener('click', resetGame);
 document.querySelector('.start-button').addEventListener('click', startGame);
 // document.querySelector('.player-name-inputs').forEach((playerName) => {
 //   playerName.on('input', checkInputs);
 // });
-// document.querySelector('.submit-dd-wager-button').addEventListener('click', submitDailyDoubleWager);
-// document.querySelector('.wager-dd-text-input').addEventListener('input', checkInputs);
+document.querySelector('.submit-dd-wager-button').addEventListener('click', submitDailyDoubleWager);
 // document.querySelector('.submit-wagers-button').addEventListener('click', submitFinalWagers);
 // document.querySelector('.wager-text-input').forEach((wager) => {
 //   wager.addEventListener('input', checkInputs);
 // });
-// document.querySelector('.submit-answer-button').addEventListener('click', submitAnswer);
-// document.querySelector('.answer-text-input').addEventListener('input', checkInputs);
+document.querySelector('.submit-answer-button').addEventListener('click', submitAnswer);
 document.querySelectorAll('.clue-box').forEach((box) => {
   box.addEventListener('click', displayClue);
 });
 
 function displayClue() {
   currentClue = currentRoundObj.clues[parseInt(event.target.dataset.id)];
-  // check if daily double
+  if (currentClue.dailyDouble) {
+  domUpdates.changeDisplayedScreen(document.querySelector('.daily-double-pop-up'));
+  }
   domUpdates.editClueHTML(currentClue, document.querySelector('.clue-text'));
   domUpdates.changeDisplayedScreen(document.querySelector('.clue-pop-up'));
 }
 
+function submitDailyDoubleWager() {
+  let dailyDouble = new DailyDouble();
+  // update highest score on board!
+  let validWager = dailyDouble.validateWager(ddWagerInput.value, 400, game.playerTurn);
+  domUpdates.updateWagerMessage(validWager);
+}
+
+function submitAnswer() {
```

Comment:
Same as above, but I wrote out code for this function in index.js file and the Clue class. Lets discuss!